amkotlinutil/build.gradle

```diff
@@ -48,30 +49,36 @@ android {
 }
 
 dependencies {
-    compileOnly 'androidx.appcompat:appcompat:1.0.2'
-    compileOnly 'com.google.android.material:material:1.1.0-alpha07'
+    compileOnly 'androidx.appcompat:appcompat:1.1.0'
+    compileOnly 'com.google.android.material:material:1.1.0-alpha10'
     compileOnly "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
     compileOnly 'androidx.constraintlayout:constraintlayout:2.0.0-beta2'
```

Comment:
Don't know if we should downgrade this to beta1, since we had problems with it in other projects?

Reply:
Good point 👍